src/client/render/plain.cpp

```diff
@@ -52,6 +52,7 @@ void DrawHUD::run(PipelineContext &context)
 		if (context.draw_crosshair)
 			context.hud->drawCrosshair();
 
+		context.hud->resizeHotbar();
 		context.hud->drawHotbar(context.client->getEnv().getLocalPlayer()->getWieldIndex());
```

Comment:
it's not possible to enforce resize on drawHotbar function
 directly if it's mandatory ?

Reply:
I've moved it before drawCrosshair() call because otherwise crosshair is jumping around on screen when using hotkeys to resize the window (e.g. maximize/minimize).